ixwebsocket/IXSocketOpenSSL.cpp

```diff
@@ -77,6 +83,43 @@ namespace
 
         return true;
     }
+    
```

Comment:
Can you add unittest for this new function ?

Reply:
I guess so, where should I add this?  Is is the reason I got message that test failed under linux, linux_asan, and mac_tsan... ?